google/cloud/forseti/common/util/date_time.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common utility functions regarding date and time."""
+
+from datetime import datetime
+
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.common.util import string_formats
+
+LOGGER = logger.get_logger(__name__)
+
+
+class UtilBaseDateTimeException(Exception):
+    """A Base UtilDateTime Exception."""
+
+
+class UtilDateTimeValueError(UtilBaseDateTimeException):
+    """Invalid Value Given for a Request."""
+
+
+class UtilDateTimeTypeError(UtilBaseDateTimeException):
```

Comment:
How about leaving `Util` out from a prefix naming?  Seems not necessary.  Do you forsee any other datetime errors to disambiguate from?

Reply:
I'd leave it but, I'll change it based on the naming request above.